checker/main.go

```diff
@@ -191,7 +191,7 @@ func checkRuleAttributeNotEmpty(ruleName, attribName, value string) {
 
 // Error code content checks.
 
-func checkErrorCodeFileNotEmpty(ruleName, errorCode, fileName string, value []byte) {
+func checkErrorCodeFileNotEmpty(ruleName, errorCode, fileName string, value string) {
 	checkStringNotEmpty(
 		fmt.Sprintf("content file '%s' of error code '%s|%s'", fileName, ruleName, errorCode),
 		string(value),
```

Comment:
imho not needed

Reply:
Some of the linters doesn't like the `[]byte`, as the `content` types changed... but I should update the line 197, using `string` is not needed anymore